actions/shared/helm_deploy/README.md

```diff
@@ -0,0 +1,53 @@
+# 🚀 Helm Deploy GitHub Action  
+This Action automates Helm install/upgrade services using Helm
+## Features
+- Checkout service and pipeline repositories
+- Namespace creation for new installations
+- Creation necessary resources before restricted installation
+- Replace Docker image versions in Helm charts for specified components
+- Deploy service with specified templates  
+- Deploy (install/upgrade) service using Helm
+- Deploy with restricted permissions or cluster-admin rights
+
+## 📌 Inputs
+
+| Name               | Description                                                                | Required | Default   | Type    |
+|--------------------|----------------------------------------------------------------------------|----------|-----------|---------|
```

Comment:
In general, it would be better to put shorter (fixed-with) columns first. Currently it is hard to tell the type and the default value for each column

Reply:
column Description moved to the right